### DISCUSSION TITLE
How to deploy?

### DISCUSSION THREAD

**Discussion by bilogic** [Z3d0X/filament-fabricator#181] [Q&A] [answered]
1. We might build the frontend in dev/stage. How does this package handle deployment to production?
2. Probably a related concern is how to backup and restore? (And no, I can't backup/restore the entire database else it will affect other data)

**Comment by Voltra** [comment, marked as answer]
For (what I'm assuming is what you want as) SSG, you can use [`spatie/laravel-export`](https://packagist.org/packages/spatie/laravel-export) (or my fork [`voltra/laravel-export`](https://packagist.org/packages/voltra/laravel-export))

As for backups, I recommend using [`spatie/laravel-backup`](https://packagist.org/packages/spatie/laravel-backup) as it's heavily customizable and can suit basically any need. There is a filament integration package which is [`shuvroroy/filament-spatie-laravel-backup`](https://packagist.org/packages/shuvroroy/filament-spatie-laravel-backup)
